### ISSUE TITLE
chromium-browser-stable crashes in VirtualBox

### ISSUE THREAD

**Issue by benbullard79** [OpenMandrivaAssociation/distribution#2818]
**OpenMandriva version**:

Cooker, Rolling

**Describe the bug**:

Latest versions (102 & 103) including `chromium-browser-stable-103.0.5060.134-1` crash in VirtualBox. Crash meaning that browser does not open at all.

**Steps to reproduce**:

In VirtualBox Cooker or Rolling system install most recent `chromium-browser-stable`. Then try to open and use browser.

**Observed behavior**:

Browser does not open.

**Expected behavior**:

That browser opens and is usable.

**Additional comment**:

I have verified this issue in Cooker and Rolling x86_64 and znver1 VBox VM's.

This looks like in *might* be the problem:

```
[1369:1410:0730/115947.090506:ERROR:ssl_client_socket_impl.cc(996)] handshake failed; returned -1, SSL error code 1, net_error -3
[1369:1410:0730/115947.091026:ERROR:ssl_client_socket_impl.cc(996)] handshake failed; returned -1, SSL error code 1, net_error -3
[1369:1410:0730/115947.091335:ERROR:ssl_client_socket_impl.cc(996)] handshake failed; returned -1, SSL error code 1, net_error -3
[1369:1410:0730/115947.091629:ERROR:ssl_client_socket_impl.cc(996)] handshake failed; returned -1, SSL error code 1, net_error -3
[1369:1410:0730/115947.092607:ERROR:ssl_client_socket_impl.cc(996)] handshake failed; returned -1, SSL error code 1, net_error -3
[1369:1410:0730/115947.093128:ERROR:ssl_client_socket_impl.cc(996)] handshake failed; returned -1, SSL error code 1, net_error -3
Trace/breakpoint trap (core dumped)
```

**Logs and screenshots if relevant**

Konsole output of attempt to open `chromium-browser-stable` in VirtualBox Cooker system:

[chromium-browser-stable.txt](https://github.com/OpenMandrivaAssociation/distribution/files/9226973/chromium-browser-stable.txt)

Bug report log of that VBox Cooker system:

[omdv-bug-report-20220730120953.log](https://github.com/OpenMandrivaAssociation/distribution/files/9226974/omdv-bug-report-20220730120953.log)

Host system is up to date Cooker/znver1 (hardware) system using `virtualbox-6.1.36-1`.

**Comment by omakuta** [comment]
This issue has been mentioned on **OMA forum**. There might be relevant details there:

https://forum.openmandriva.org/t/chromium-browser-stable-103-broken-in-virtualbox/4486/5

**Comment by benbullard79** [comment]
AngryPenguinPL told that this is probably related to something in vaapi. But he does not know how to disable whatever needs disabling for the browser to open in VBox.

**There is a work around:**

Start browser from Konsole:

`chromium-browser-stable --disable-gpu`

That should open the browser. Then in open browser go to Settings>System and disable hardware acceleration. Now browser should open normally in VirtualBox.

**Comment by edwardp3** [comment]
The browser will launch with hardware acceleration disabled, but it will run for roughly 10 seconds, freeze, then crash.

**Comment by edwardp3** [comment]
Attempted to try Chromium again in a VirtualBox, except with 3D Acceleration enabled. Made one change to a Google-related setting, which required a restart. The restart ended in a black browser screen. Desktop was Plasma/X11.

This screenshot was taken from the host OS, because `screengrab` would not launch after it was also installed, will be filing a bug report for that.

![chromium-openmandriva](https://user-images.githubusercontent.com/73129450/185437854-32505d86-23db-4b30-a720-2400994cc5da.png)

**Comment by edwardp3** [comment]
Chromium produced two crashes in VirtualBox (Cooker) when it crashes on the Google Web Store URL. These are the `coredumpctl debug` outputs from both.

[chromium-crash-1.txt](https://github.com/OpenMandrivaAssociation/distribution/files/9481462/chromium-crash-1.txt)
[chromium-crash-2.txt](https://github.com/OpenMandrivaAssociation/distribution/files/9481464/chromium-crash-2.txt)

**Comment by edwardp3** [comment]
I noticed a new version in the Rolling repository, 105.0.5195.102. Installed on OM Rolling on-hardware (no VirtualBox) and it also crashed on the Google Web Store home page.

**Comment by edwardp3** [comment]
105.0.5195.102 also crashes in Cooker inside a VirtualBox.

**Comment by omakuta** [comment]
This issue has been mentioned on **OMA forum**. There might be relevant details there:

https://forum.openmandriva.org/t/chromium-browser-stable-broken-in-virtualbox/4486/12

**Comment by edwardp3** [comment]
Installed the latest version in Rolling, then ran it from the terminal. Received the following output.

`coredumpctl` displays four different crashes for the browser.

```
~]$ chromium-browser-stable
[1607:1607:0925/152031.092845:FATAL:spawn_subprocess.cc(221)] posix_spawn: No such file or directory (2)
#0 0x56194ee7c8b7 (/usr/lib64/chromium-browser-stable/chrome+0x9e178b6)
#1 0x56194ee7e2fe (/usr/lib64/chromium-browser-stable/chrome+0x9e192fd)
#2 0x561952025bfc (/usr/lib64/chromium-browser-stable/chrome+0xcfc0bfb)
#3 0x5619520222ee (/usr/lib64/chromium-browser-stable/chrome+0xcfbd2ed)
#4 0x56195203d189 (/usr/lib64/chromium-browser-stable/chrome+0xcfd8188)
#5 0x56195201f138 (/usr/lib64/chromium-browser-stable/chrome+0xcfba137)
#6 0x56194910770a (/usr/lib64/chromium-browser-stable/chrome+0x40a2709)
#7 0x56194e75dd1e (/usr/lib64/chromium-browser-stable/chrome+0x96f8d1d)
#8 0x56194e75b3b8 (/usr/lib64/chromium-browser-stable/chrome+0x96f63b7)
#9 0x56194e75b7a6 (/usr/lib64/chromium-browser-stable/chrome+0x96f67a5)
#10 0x561949105bd4 (/usr/lib64/chromium-browser-stable/chrome+0x40a0bd3)
#11 0x7fb03e3021ca (/usr/lib64/libc.so.6+0x231c9)
#12 0x7fb03e302285 (/usr/lib64/libc.so.6+0x23284)
#13 0x561948d45021 (/usr/lib64/chromium-browser-stable/chrome+0x3ce0020)

[1619:1619:0925/152032.667137:ERROR:data_pack.cc(221)] Failed to open datapack with base::File::Error -4
[1:1:0925/152032.688055:ERROR:data_pack.cc(221)] Failed to open datapack with base::File::Error -4
[1604:1604:0925/152032.703882:ERROR:data_pack.cc(221)] Failed to open datapack with base::File::Error -4
libEGL warning: DRI2: failed to authenticate
[1644:1644:0925/152033.203167:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1644:1644:0925/152033.205966:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[1644:1644:0925/152033.209313:ERROR:gpu_init.cc(486)] Passthrough is not supported, GL is angle, ANGLE is swiftshader
[1644:1644:0925/152033.215159:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1644:1644:0925/152033.215725:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
[1644:1644:0925/152033.216067:ERROR:gpu_info_collector.cc(93)] gl::init::CreateGLContext failed
[1644:1644:0925/152033.221207:ERROR:gpu_info_collector.cc(354)] Could not create context for info collection.
[1644:1644:0925/152033.222709:ERROR:gpu_init.cc(86)] CollectGraphicsInfo failed.
[1644:1644:0925/152033.267004:ERROR:viz_main_impl.cc(186)] Exiting GPU process due to errors during initialization
[1645:1645:0925/152033.356682:ERROR:data_pack.cc(221)] Failed to open datapack with base::File::Error -4
[1668:1:0925/152033.793168:ERROR:runtime_enabled_features.cc(2440)] RuntimeEnabledFeature not recognized: TrustedDOMTypes
[1669:1:0925/152033.825243:ERROR:runtime_enabled_features.cc(2440)] RuntimeEnabledFeature not recognized: TrustedDOMTypes
libEGL warning: DRI2: failed to authenticate
[1696:1:0925/152034.391799:ERROR:runtime_enabled_features.cc(2440)] RuntimeEnabledFeature not recognized: TrustedDOMTypes
[1689:1689:0925/152034.591766:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1689:1689:0925/152034.593056:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
[1703:1:0925/152034.592816:ERROR:runtime_enabled_features.cc(2440)] RuntimeEnabledFeature not recognized: TrustedDOMTypes
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[1689:1689:0925/152034.625559:ERROR:gpu_init.cc(486)] Passthrough is not supported, GL is angle, ANGLE is swiftshader
[1689:1689:0925/152034.651698:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1689:1689:0925/152034.664324:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
[1689:1689:0925/152034.664590:ERROR:gpu_info_collector.cc(93)] gl::init::CreateGLContext failed
[1689:1689:0925/152034.664699:ERROR:gpu_info_collector.cc(354)] Could not create context for info collection.
[1689:1689:0925/152034.664864:ERROR:gpu_init.cc(86)] CollectGraphicsInfo failed.
[1689:1689:0925/152034.767295:ERROR:viz_main_impl.cc(186)] Exiting GPU process due to errors during initialization
libEGL warning: DRI2: failed to authenticate
[1719:1719:0925/152035.168149:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1719:1719:0925/152035.168963:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[1719:1719:0925/152035.173845:ERROR:gpu_init.cc(486)] Passthrough is not supported, GL is angle, ANGLE is swiftshader
[1719:1719:0925/152035.191821:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1719:1719:0925/152035.192179:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
[1719:1719:0925/152035.192420:ERROR:gpu_info_collector.cc(93)] gl::init::CreateGLContext failed
[1719:1719:0925/152035.192597:ERROR:gpu_info_collector.cc(354)] Could not create context for info collection.
[1719:1719:0925/152035.192824:ERROR:gpu_init.cc(86)] CollectGraphicsInfo failed.
[1719:1719:0925/152035.274016:ERROR:viz_main_impl.cc(186)] Exiting GPU process due to errors during initialization
libEGL warning: DRI2: failed to authenticate
[1731:1731:0925/152035.975511:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1731:1731:0925/152035.975872:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[1731:1731:0925/152035.981888:ERROR:gpu_init.cc(486)] Passthrough is not supported, GL is angle, ANGLE is swiftshader
[1731:1731:0925/152035.993548:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1731:1731:0925/152035.994294:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
[1731:1731:0925/152035.995481:ERROR:gpu_info_collector.cc(93)] gl::init::CreateGLContext failed
[1731:1731:0925/152035.995652:ERROR:gpu_info_collector.cc(354)] Could not create context for info collection.
[1731:1731:0925/152035.996135:ERROR:gpu_init.cc(86)] CollectGraphicsInfo failed.
[1731:1731:0925/152036.047259:ERROR:viz_main_impl.cc(186)] Exiting GPU process due to errors during initialization
libEGL warning: DRI2: failed to authenticate
[1739:1739:0925/152036.345185:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1739:1739:0925/152036.346413:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[1739:1739:0925/152036.349513:ERROR:gpu_init.cc(486)] Passthrough is not supported, GL is angle, ANGLE is swiftshader
[1739:1739:0925/152036.354522:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1739:1739:0925/152036.354853:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
[1739:1739:0925/152036.355342:ERROR:gpu_info_collector.cc(93)] gl::init::CreateGLContext failed
[1739:1739:0925/152036.355594:ERROR:gpu_info_collector.cc(354)] Could not create context for info collection.
[1739:1739:0925/152036.473667:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1739:1739:0925/152036.473949:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
[1739:1739:0925/152036.474264:ERROR:gpu_channel_manager.cc(828)] Failed to create GLES3 context, fallback to GLES2.
[1739:1739:0925/152036.474574:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1739:1739:0925/152036.474783:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
[1739:1739:0925/152036.474878:ERROR:gpu_channel_manager.cc(839)] ContextResult::kFatalFailure: Failed to create shared context for virtualization.
[1739:1739:0925/152036.474981:ERROR:shared_image_stub.cc(489)] SharedImageStub: unable to create context
[1739:1739:0925/152036.475309:ERROR:gpu_channel.cc(608)] GpuChannel: Failed to create SharedImageStub
[1739:1739:0925/152036.479241:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1739:1739:0925/152036.479587:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
[1739:1739:0925/152036.479855:ERROR:gpu_channel_manager.cc(828)] Failed to create GLES3 context, fallback to GLES2.
[1739:1739:0925/152036.481657:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1739:1739:0925/152036.481854:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
[1739:1739:0925/152036.481956:ERROR:gpu_channel_manager.cc(839)] ContextResult::kFatalFailure: Failed to create shared context for virtualization.
[1739:1739:0925/152036.482058:ERROR:shared_image_stub.cc(489)] SharedImageStub: unable to create context
[1739:1739:0925/152036.482277:ERROR:gpu_channel.cc(608)] GpuChannel: Failed to create SharedImageStub
[1739:1739:0925/152036.482749:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1739:1739:0925/152036.482883:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
[1739:1739:0925/152036.483094:ERROR:gpu_channel_manager.cc(828)] Failed to create GLES3 context, fallback to GLES2.
[1739:1739:0925/152036.483309:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1739:1739:0925/152036.483565:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
[1739:1739:0925/152036.483793:ERROR:gpu_channel_manager.cc(839)] ContextResult::kFatalFailure: Failed to create shared context for virtualization.
[1739:1739:0925/152036.483984:ERROR:shared_image_stub.cc(489)] SharedImageStub: unable to create context
[1739:1739:0925/152036.486853:ERROR:gpu_channel.cc(608)] GpuChannel: Failed to create SharedImageStub
[1739:1739:0925/152036.487224:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1739:1739:0925/152036.487449:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
[1739:1739:0925/152036.490007:ERROR:gpu_channel_manager.cc(828)] Failed to create GLES3 context, fallback to GLES2.
[1739:1739:0925/152036.491488:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1739:1739:0925/152036.491645:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
[1739:1739:0925/152036.491860:ERROR:gpu_channel_manager.cc(839)] ContextResult::kFatalFailure: Failed to create shared context for virtualization.
[1739:1739:0925/152036.492059:ERROR:shared_image_stub.cc(489)] SharedImageStub: unable to create context
[1739:1739:0925/152036.492277:ERROR:gpu_channel.cc(608)] GpuChannel: Failed to create SharedImageStub
[1739:1739:0925/152036.493024:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1739:1739:0925/152036.493192:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
[1739:1739:0925/152036.493549:ERROR:gpu_channel_manager.cc(828)] Failed to create GLES3 context, fallback to GLES2.
[1739:1739:0925/152036.493848:ERROR:gl_display.cc(506)] EGL Driver message (Error) eglCreateContext: eglCreateContext
[1739:1739:0925/152036.494143:ERROR:gl_context_egl.cc(358)] eglCreateContext failed with error EGL_SUCCESS
[1739:1739:0925/152036.494465:ERROR:gpu_channel_manager.cc(839)] ContextResult::kFatalFailure: Failed to create shared context for virtualization.
[1739:1739:0925/152036.494766:ERROR:shared_image_stub.cc(489)] SharedImageStub: unable to create context
[1739:1739:0925/152036.495052:ERROR:gpu_channel.cc(608)] GpuChannel: Failed to create SharedImageStub
[1604:1604:0925/152036.484893:FATAL:gpu_client.cc(107)] Check failed: channel_handle.is_valid() == status == GpuHostImpl::EstablishChannelStatus::kSuccess (0 vs. 1)
#0 0x56194ee7c8b7 logging::LogMessage::~LogMessage()
#1 0x56194ee7e089 logging::LogMessage::~LogMessage()
#2 0x561950d8f21e viz::GpuClient::OnEstablishGpuChannel()
#3 0x561950d9065b base::internal::Invoker<>::RunOnce()
#4 0x561950d93fbb viz::GpuHostImpl::OnChannelEstablished()
#5 0x561950d96a12 base::internal::Invoker<>::RunOnce()
#6 0x56194aa49ce7 viz::mojom::GpuService_EstablishGpuChannel_ForwardToCallback::Accept()
#7 0x56194f33c505 mojo::InterfaceEndpointClient::HandleIncomingMessageThunk::Accept()
#8 0x56194f347a61 mojo::MessageDispatcher::Accept()
#9 0x56194f33eb4b mojo::InterfaceEndpointClient::HandleIncomingMessage()
#10 0x56194f34fe12 mojo::internal::MultiplexRouter::ProcessIncomingMessage()
#11 0x56194f34edc9 mojo::internal::MultiplexRouter::Accept()
#12 0x56194f347a61 mojo::MessageDispatcher::Accept()
#13 0x56194f33839b mojo::Connector::DispatchMessage()
#14 0x56194f338f64 mojo::Connector::ReadAllAvailableMessages()
#15 0x56194f338c30 mojo::Connector::OnWatcherHandleReady()
#16 0x56194f372bd5 mojo::SimpleWatcher::OnHandleReady()
#17 0x56194f372ec5 base::internal::Invoker<>::RunOnce()
#18 0x56194eee4bcc base::TaskAnnotator::RunTaskImpl()
#19 0x56194ef1b822 base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWork()
#20 0x56194ee88827 base::MessagePumpGlib::Run()
#21 0x56194ef1d2b4 base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::Run()
#22 0x56194eebe5fd base::RunLoop::Run()
#23 0x56194b237264 content::BrowserMainLoop::RunMainMessageLoop()
#24 0x56194b239dd0 content::BrowserMainRunnerImpl::Run()
#25 0x56194b233bed content::BrowserMain()
#26 0x56194e75eee7 content::ContentMainRunnerImpl::RunBrowser()
#27 0x56194e75e9e6 content::ContentMainRunnerImpl::Run()
#28 0x56194e75b5ef content::RunContentProcess()
#29 0x56194e75b7a6 content::ContentMain()
#30 0x561949105bd4 ChromeMain
#31 0x7fb03e3021ca (/usr/lib64/libc.so.6+0x231c9)
#32 0x7fb03e302285 __libc_start_main
#33 0x561948d45021 _start
Task trace:
#0 0x56194f371487 mojo::SimpleWatcher::Context::CallNotify()
Crash keys:
  "gpu-generation-intel" = "0"
  "gpu-devid" = "0x0405"
  "gpu-venid" = "0x15ad"
  "ui_scheduler_async_stack" = "0x56194F371487 0x0"
  "num-extensions" = "0"
  "breadcrumbs" = "0:00:02 Tab1 PageLoad
0:00:02 Tab1 FinishNav4
0:00:02 Tab1 StartNav4 #renderer-script #auto_subframe
0:00:02 Tab1 FinishNav1
0:00:01 Tab1 AddInfobar44
0:00:00 Browser1 Insert active Tab1 at 0
0:00:00 Tab1 StartNav1 #ntp #auto_toplevel
0:00:00 Startup
"
  "io_scheduler_async_stack" = "0x56194F371487 0x0"
  "variations" = "5f2c0f7c-3f4a17df,7d74eac0-7d74eac0,69d4ebd5-3f4a17df,2510663e-3f4a17df,391562d6-3f4a17df,fc72a64b-3f4a17df,e28cd73c-3f4a17df,36d5ee52-3f4a17df,65570806-377be55a,ade3efeb-e1cc0f14,3fd33f16-da51fc56,1166396-1166396,313957be-3ef44cd2,5f36436a-f799c15e,4852ec7f-3f4a17df,baee3c29-3f4a17df,250dda8b-3f4a17df,255dfea8-cf12f279,826f3281-92742f4b,df4ed2c4-aa3d81ae,ef4764d7-c9f4d4ef,2da2abac-b7f59038,a779bb20-3f4a17df,34a9ddc3-54a64e37,d3566fbd-c6f74b94,4d40c903-20e31142,18324944-3f4a17df,8b5e9272-5124c949,1d0518a-3f4a17df,ad4acdda-3f4a17df,aa9914c2-3f4a17df,97023bae-3f4a17df,b7a22696-2cd5fcd8,931c5f72-3f4a17df,b1ceb06f-3f4a17df,2856aa31-3f4a17df,a2fd384c-cc71bb94,1fce7d57-3f4a17df,e8c68789-49a20295,b53f3ef9-3f4a17df,d990c4ac-3f4a17df,a18444ea-a18444ea,6a24725f-44b296ce,3482a891-410c5d63,29c62d4-3f4a17df,4d936449-fd549ada,9e5c75f1-30e1b12b,a7d30c71-5124c949,62cb9b8b-41986bdd,dbf7a8af-3f4a17df,eddd0d82-3f4a17df,863ba1ea-3f4a17df,df319cb2-27e0e1e8,4749874c-455e925b,caa76e48-caa76e48,25ad6029-3f4a17df,b0f15b33-b0f15b33,e79de56c-dee0823,7760b5b2-3f4a17df,5c4d440e-ed0e492d,2fb38473-3f4a17df,234de0a0-ace4e138,13e2821-b8f74491,8d7344de-3f4a17df,683e86a8-3f4a17df,e5726113-3f4a17df,bd717e5b-3f4a17df,13400df3-3f4a17df,6cbcf5b7-3f4a17df,cae31965-2f8546fe,8bccc03b-3f4a17df,e2174ff9-3f4a17df,6e69a63-3f4a17df,2aed9870-19257351,1bb6a450-3f4a17df,e9e64bcb-3f4a17df,ef1cfe77-3f4a17df,363012bc-3f4a17df,483d5a5e-d379aaa3,a79ba57a-3d47f4f4,4949c1d8-3f4a17df,210a630a-3f4a17df,9f7bb00b-3f4a17df,55c6e56f-a1ba80e8,f6e27768-f695fd79,3b96a1d-3f4a17df,6becb1e-a6ea97a2,595f5eb0-f23d1dea,dba92675-f23d1dea,376d29ba-3f4a17df,c8a4c657-c8a4c657,4ea303a6-3f4a17df,3042ad4b-ad2fa222,569920b-89a4b7af,42f0e0ea-75d6947c,bde7927d-bab20a64,4e3ec83a-e4938e2c,645ea67f-5de8c8da,5306c29b-9cbf73ff,f26e9e11-3f4a17df,d7f231f3-6e2eda13,1130dae-ccb721cc,7d66e042-3f4a17df,3e7d7783-f38a9353,53879ff4-3f4a17df,a393b0ae-14f93b4e,44c2e2c0-3f4a17df,74f8fa8f-74f8fa8f,653036a-3f4a17df,55ba4cfa-3f4a17df,9481ce98-3d47f4f4,4b935545-3d47f4f4,70678518-dee66fa8,9a38bae3-3d47f4f4,122c746b-3d47f4f4,d69d967d-3695c92e,"
  "num-experiments" = "116"
  "switch-1" = "--enable-crashpad"
  "num-switches" = "3"
  "osarch" = "x86_64"
  "pid" = "1604"
  "ptype" = "browser"

[1604:1604:0925/152037.411464:ERROR:socket.cc(93)] sendmsg: Broken pipe (32)
Trace/breakpoint trap (core dumped)
```

**Comment by edwardp3** [comment]
Running `chromium-browser-stable %u --disable-gpu` or `chromium-browser-stable --disable-gpu ` produces essentially the same output as above. 

Although it seems to work without crashing when installed in my Xfce Cooker VBox, editing the .desktop file to add the --disable-gpu flag to the command.

**Comment by benbullard79** [comment]
This is fixed with `chromium-browser-stable-107.0.5304.110-2` in Cooker and ROME.
